src/components/app.js

```diff
@@ -201,17 +292,23 @@ let App = class App extends React.PureComponent {
     const { connectDropTarget, isOver } = this.props;
 
     return connectDropTarget(
-      <div onMouseMove={this.trackCoordinates}>
-        <div className="flex-parent flex-parent--end-cross flex-parent--center-main absolute top right bottom left">
-          <div className="flex-child mb24 z1 txt-s txt-bold flex-parent">
-            <div className="flex-child bg-darken75 color-white inline-block pl24 pr12 py12 round-l-full">
-              {helpText}
+      <div>
+        <div className="sliderHolder">
+          <ReactSlider orientation={'vertical'} invert={true} defaultValue={300} min={250} max={5000} onChange={this.updateHandle.bind(this)} onAfterChange={this.sendValue.bind(this)} />
```

Comment:
Curious why `bind` is needed. Does ReactSlider change the context of `this` to the DOM element?

Reply:
Ah yeah, I tried without and it isn't. I haven't worked with React very much, so I was just going by examples I'd see. Totally works without and has been removed now.